### PR TITLE
Update runtime to gnome44

### LINKED ISSUE #19: Remove dconf workaround
https://blogs.gnome.org/mclasen/2019/07/12/settings-in-a-sandbox-world/

### PR DESCRIPTION
**Pull request by hfiguiere**
- Added libsoup-2.4 to the build
- Added opencv patch
- Did the long overdue dconf migration (Close #19)

### CODE REVIEW

**Comment by aleb** [comment]
Thanks! What's the story of the opencv patch?

**Comment by hfiguiere** [comment]
that opencv use `-Werrror` for non development builds beside this being a bad idea. And CMake doesn't seem to honour the cflags.

**Comment by hfiguiere** [comment]
Without the patch:
```
[ 44%] Built target IlmImf
cc1plus: warning: command-line option ‘-Wmissing-prototypes’ is valid for C/ObjC but not for C++
cc1plus: warning: command-line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
/run/build/opencv/modules/core/src/persistence_base64.cpp: In function ‘bool base64::base64_valid(const uint8_t*, size_t, size_t)’:
/run/build/opencv/modules/core/src/persistence_base64.cpp:167:31: error: comparing the result of pointer addition ‘(src + ((sizetype)off))’ and NULL [-Werror=address]
  167 |     if (src == 0 || src + off == 0)
      |                     ~~~~~~~~~~^~~~
cc1plus: some warnings being treated as errors
make[2]: *** [modules/core/CMakeFiles/opencv_core.dir/build.make:943: modules/core/CMakeFiles/opencv_core.dir/src/persistence_base64.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
```

**Comment by hfiguiere** [comment]
Updated to remove the extra build options as the flags are ignored by CMake.

**Comment by aleb** [comment]
Ok to merge?

**Comment by hfiguiere** [comment]
yes

**Comment by aleb** [comment]
Thanks a lot!

File: org.pitivi.Pitivi.json [hunk 1]
```diff
@@ -350,12 +351,12 @@
         },
         "shared-modules/linux-audio/lv2.json",
         "shared-modules/linux-audio/lilv.json",
+        "shared-modules/libsoup/libsoup-2.4.json",
         {
             "name": "gstreamer",
             "buildsystem": "meson",
             "builddir": true,
             "config-opts": [
-                "-Dpygi-overrides-dir=/app/lib/python3.9/site-packages/gi/overrides/",
```
Comment (aleb) [inline org.pitivi.Pitivi.json:358]: In the dev sandbox we have `"-Dgst-python:pygi-overrides-dir=/app/lib/python3.10/site-packages/gi/overrides/",` shall we add it also here  @thiblahute ?

Comment (hfiguiere) [reply org.pitivi.Pitivi.json:358]: it triggered a build error to have that option.

Comment (hfiguiere) [reply org.pitivi.Pitivi.json:358]: I mean the one I removed. (it was set to python3.10)

**Pull request merged**